website/content/docs/agent/options.mdx

```diff
@@ -642,19 +642,25 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
   - `tokens` ((#acl_tokens)) - This object holds all of the configured
     ACL tokens for the agents usage.
 
-    - `master` ((#acl_tokens_master)) - Only used for servers in the [`primary_datacenter`](#primary_datacenter).
-      This token will be created with management-level permissions if it does not
-      exist. It allows operators to bootstrap the ACL system with a token Secret
-      ID that is well-known.
-
-      The `master` token is only installed when a server acquires cluster
-      leadership. If you would like to install or change the `acl_master_token`,
-      set the new value for `master` in the configuration for all servers. Once
-      this is done, restart the current leader to force a leader election. If
-      the `master` token is not supplied, then the servers do not create a
-      master token. When you provide a value, it should be a UUID. To maintain
-      backwards compatibility and an upgrade path this restriction is not
-      currently enforced but will be in a future major Consul release.
+    - `initial_management` ((#acl_tokens_initial_management)) - This is available in
+      Consul 1.11 and later. In prior versions, use [`acl.tokens.master`](#acl_tokens_master).
+
+      Only used for servers in the [`primary_datacenter`](#primary_datacenter).
+      This token will be created with management-level permissions if it does not exist.
+      It allows operators to bootstrap the ACL system with a token Secret ID that is
+      well-known.
+
+      The `initial_management` token is only installed when a server acquires cluster
+      leadership. If you would like to install or change it, set the new value for
+      `initial_management` in the configuration for all servers. Once this is done,
+      restart the current leader to force a leader election. If the `initial_management`
+      token is not supplied, then the servers do not create an initial management token.
+      When you provide a value, it should be a UUID. To maintain backwards compatibility
+      and an upgrade path this restriction is not currently enforced but will be in a
+      future major Consul release.
+
+    - `master` ((#acl_tokens_master)) **Deprecated in Consul 1.11. Use the
+      [`acl.tokens.initial_management`](#acl_tokens_initial_management) field instead.**
```

Comment:
For someone on Consul <1.11, I'm not sure this text makes it clear that the `initial_management` token is a direct replacement (a rename), so the docs for `initial_management` apply here, too.

Perhaps this? Let me know what you think.
```suggestion
    - `master` ((#acl_tokens_master)) **Renamed in Consul 1.11 to
      [`acl.tokens.initial_management`](#acl_tokens_initial_management).**
```

Or if we **must** include the word deprecated...
```suggestion
    - `master` ((#acl_tokens_master)) **Deprecated in Consul 1.11: renamed to
      [`acl.tokens.initial_management`](#acl_tokens_initial_management).**
```

Reply:
I agree, rename is clearer 👍🏻